WeakAuras/Prototypes.lua

```diff
@@ -9903,6 +9909,25 @@ Private.event_prototypes = {
         enable = WeakAuras.IsRetail(),
         hidden = not WeakAuras.IsRetail(),
       },
+      {
+        name = "loot_specialization_id",
+        display = L["Loot Specialization"],
+        type = "multiselect",
+        init = "GetLootSpecialization() == 0 and GetSpecializationInfo(GetSpecialization()) or GetLootSpecialization()",
+        values = "spec_types_all",
+        enable = WeakAuras.IsRetail(),
+        hidden = not WeakAuras.IsRetail(),
+      },
+      {
+        name = "lootSpec",
+        display = L["Loot Specialization"],
+        type = "string",
+        init = "select(2, GetSpecializationInfoByID(loot_specialization_id))",
+        enable = "loot_specialization_id",
+        hidden = true,
+        store = true,
+        test = "true",
+      },
```

Comment:
this needs a 

        enable = WeakAuras.IsRetail(),
        hidden = not WeakAuras.IsRetail(),
        
getting this error on SoD
```
[string ""]:14: attempt to call global 'GetSpecializationInfoByID' (a nil value)
[string ""]:14: in function <[string ""]:2>
[string "=[C]"]: in function `xpcall'
[string "@WeakAuras/GenericTrigger.lua"]:707: in function <WeakAuras/GenericTrigger.lua:630>
[string "@WeakAuras/GenericTrigger.lua"]:1027: in function `ScanWithFakeEvent'
[string "@WeakAuras/GenericTrigger.lua"]:1370: in function `LoadDisplays'
[string "@WeakAuras/WeakAuras.lua"]:1939: in function `LoadDisplays'
[string "@WeakAuras/WeakAuras.lua"]:1723: in function <WeakAuras/WeakAuras.lua:1567>
[string "@WeakAuras/WeakAuras.lua"]:1916: in function `Resume'
[string "@WeakAurasOptions/OptionsFrames/OptionsFrame.lua"]:170: in function <...dOns/WeakAurasOptions/OptionsFrames/OptionsFrame.lua:150>
[string "=[C]"]: in function `Hide'
[string "@Interface_Vanilla/FrameXML/UIParent.lua"]:2665: in function `HideUIPanel'
[string "@SharedXML/SharedUIPanelTemplates.lua"]:213: in function `HideParentPanel'
```